### PR TITLE
[CI:DOCS] Document source ip for the rootlesskit port handler

### LINKED ISSUE #10884: Document that source IP for rootless containers is always 10.0.2.100 and slirp4netns workaround with it's (dis)advantages
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind documentation
(?)

## Problem

I had a [curious "problem" with the webserver caddy](https://caddy.community/t/use-real-ip-in-docker-podman-container/12978?u=rugk), where each source IP was always `10.0.2.100`. I could trace it back to podman being the obvious fault, because it's the "inside of container" vs "outside of container" view… :upside_down_face: 
In the end, I solved this by switching to `host` networking…

Later I noticed this comment by @AkihiroSuda:
> Also it should be documented that apps will always see 10.0.2.100 (or CNI IP) as the source address.

https://github.com/containers/podman/pull/9052#issuecomment-764700524

## Documentation

[Searching the docs](http://docs.podman.io/en/latest/search.html?q=10.0.2.100) it seems that this comment was forgotten and nothing was actually documented.

As such, here is an issue to document the fact that this happens and explain the workaround/solution in this case (which AFAIK are either using rootfull containers or host networking as I did).
**Edit:** or… `--network slirp4netns:port_handler=slirp4netns`, apparently, [as I was made ware](https://caddy.community/t/use-real-ip-in-docker-podman-container/12978/5?u=rugk), ref https://github.com/containers/podman/pull/9052 (anyway, that also needs documentation)

## Relevance

IMHO this is important to document, because the use case of having a proper ingoing IP address is quite popular for all webservers, I guess.
Also, it's a difference to Docker/limitation of rootless containers, so IMHO, it [should also be documented here](https://github.com/containers/podman/blob/main/rootless.md).

### PR DESCRIPTION
**Pull request by Luap99**
Fixes #10884

### CODE REVIEW

File: docs/source/markdown/podman-create.1.md [hunk 1]
```diff
@@ -678,7 +678,8 @@ Valid _mode_ values are:
   - **outbound_addr6=INTERFACE**: Specify the outbound interface slirp should bind to (ipv6 traffic only).
   - **outbound_addr6=IPv6**: Specify the outbound ipv6 address slirp should bind to.
   - **port_handler=rootlesskit**: Use rootlesskit for port forwarding. Default.
-  - **port_handler=slirp4netns**: Use the slirp4netns port forwarding.
+  Note: Rootlesskit changes the source ip of incoming packets to a IP in the container network namespace, usually `10.0.2.100`. If your application requires the real source ip, e.g. web server logs, use the slirp4netns port handler. This port handler is also used for rootless users when connected to user-defined networks.
```
Comment (AkihiroSuda) [inline docs/source/markdown/podman-create.1.md:681]: > This port handler is also used for rootless users when connected to user-defined networks.

Perhaps "rootless users" -> "rootless containers"

Comment (AkihiroSuda) [reply docs/source/markdown/podman-create.1.md:681]: Also, "This port handler" -> "RootlessKit port handler" might be better. (Some people may interpret "this" to be slirp4netns that is mentioned in the previous sentence)

Comment (AkihiroSuda) [reply docs/source/markdown/podman-create.1.md:681]: nit: "ip" -> "IP address"

**Review by AkihiroSuda — APPROVED** [review]

File: docs/source/markdown/podman-pod-create.1.md [hunk 2]
```diff
@@ -102,19 +102,23 @@ Assign a name to the pod.
...
 Set network mode for the pod. Supported values are
-- **bridge**: Create a network stack on the default bridge. This is the default for rootfull containers.
-- **host**: Do not create a network namespace, all containers in the pod will use the host's network. Note: the host mode gives the container full access to local system services such as D-bus and is therefore considered insecure.
-- Comma-separated list of the names of CNI networks the pod should join.
-- **slirp4netns[:OPTIONS,...]**: use slirp4netns to create a user network stack.  This is the default for rootless containers.  It is possible to specify these additional options:
-  - **allow_host_loopback=true|false**: Allow the slirp4netns to reach the host loopback IP (`10.0.2.2`). Default is false.
+- **bridge**: create a network stack on the default bridge;
+- **none**: no networking;
+- **host**: use the Podman host network stack. Note: the host mode gives the container full access to local system services such as D-bus and is therefore considered insecure;
+- **network**: connect to a user-defined network, multiple networks should be comma-separated;
```
Comment (baude) [inline docs/source/markdown/podman-pod-create.1.md:108]: remove semi colons?

**Comment by baude** [comment]
lgtm other than the nit

File: docs/source/markdown/podman-pod-create.1.md (hunk 2 above)
Comment (Luap99) [reply docs/source/markdown/podman-pod-create.1.md:108]: I copied this from podman create. I can remove the semi colons everywhere if you want.

**Comment by mheon** [comment]
Semicolon removal seems like a good idea, otherwise LGTM

**Pull request merged**